custom-connectors/Public360/Artifacts/apiDefinition.swagger.json

```diff
@@ -0,0 +1,281 @@
+                  },
+                  "description": "parameter",
+                  "required": [
+                    "DocumentRecno",
+                    "DocumentNumber",
+                    "Format",
+                    "Title"
+                  ]
+                }
+              },
+              "default": {
+                "parameter": {
+                  "Title": "Test Case wednesday",
+                  "DocumentNumber": "20/00269-2",
+                  "DocumentRecno": "200096",
+                  "Format": "txt",
+                  "Base64Data": "test"
+                }
+              },
+              "required": [
+                "parameter"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/Biz/v2/api/call/SI.Data.RPC/SI.Data.RPC/DocumentService/CreateDocument": {
+      "post": {
+        "summary": "Create Document",
+        "description": "Create Document",
+        "operationId": "CreateDocument",
+        "parameters": [
+          {
+            "name": "hosturl",
+            "in": "query",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "parameter": {
+                  "type": "object",
+                  "properties": {
+                    "CaseNumber": {
+                      "type": "string",
+                      "description": "Case Number",
+                      "title": "Case Number"
+                    },
+                    "Title": {
+                      "type": "string",
+                      "description": "Title",
+                      "title": ""
+                    },
+                    "Status": {
+                      "type": "string",
+                      "description": "Status",
+                      "title": ""
+                    },
+                    "Category": {
+                      "type": "string",
+                      "description": "Category",
+                      "title": ""
+                    },
+                    "Notes": {
+                      "type": "string",
+                      "description": "Notes",
+                      "title": ""
+                    }
+                  },
+                  "description": "parameter",
+                  "required": [
+                    "Title",
+                    "Category",
+                    "Status",
+                    "CaseNumber"
+                  ]
+                }
+              },
+              "required": [
+                "parameter"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "default": {

```

Comment:
@Pranav2889 Please replace the `default` responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.